src/agoradatatools/process.py

```diff
@@ -127,7 +127,7 @@ def process_dataset(
             filename=dataset_name + "." + dataset_obj[dataset_name]["final_format"],
         )
 
-    gx_enabled = "gx_enabled" in dataset_obj[dataset_name].keys()
+    gx_enabled = dataset_obj[dataset_name].get("gx_enabled", False)
```

Comment:
Nit: Unsure how complicated the tests are, but this would've been a great candidate for TDD.  This was a behavior we were missing, so the test could've been added first and the test being run would fail the test.